opta/core/aws.py

```diff
@@ -435,17 +435,22 @@ def cluster_exist(self) -> bool:
         except client.exceptions.ResourceNotFoundException:
             return False
 
+    def get_kubectl_context_name(self) -> str:
```

Comment:
for the name, the kubeconfig is not only specific to kubectl, so we can just call it `kube_context`
example of another tool using it https://skaffold.dev/docs/environment/kube-context/